cmd/psa/create.go

```diff
@@ -51,23 +51,27 @@ Note that the default profile is http://arm.com/psa/2.0.0.
 				return err
 			}
 
-			claims, err := loadClaimsFromFile(fs, *createClaimsFile, false)
+			validateClaims := false
+			claims, err := loadClaimsFromFile(fs, *createClaimsFile, validateClaims)
 			if err != nil {
 				return err
 			}
 
-			evidence := psatoken.Evidence{}
-
-			if err = evidence.SetClaims(claims); err != nil {
-				return err
-			}
 			profile, err := claims.GetProfile()
 			if err != nil {
 				return fmt.Errorf("error loading profile from claims %w", err)
 			}
-			if err := matchProfile(profile); err != nil {
+			if profile != *createTokenProfile {
+				return fmt.Errorf("profile mismatch input: %s and created: %s", *createTokenProfile, profile)
```

Comment:
instead of "created" is "loaded from file" -- we don't synthesise the profile, we read what we are given, right?

Reply:
Wait this is due to name of the file "create.go" and associated to create subcommand, nothing to do with the profile been created or received..

Reply:
`profile mismatch: requested: %s, loaded: %s"`

Reply:
Thank you, corrected the code and changed the TC